### PR TITLE
Vtep clean

### PR DESCRIPTION
**Pull request by outscale-mgo**

### CODE REVIEW

File: src/vtep.c [hunk 1]
```diff
@@ -604,6 +605,81 @@ static inline int from_vtep(struct pg_brick *brick, enum pg_side from,
...
+}
+
+static inline int from_vtep(struct pg_brick *brick, enum pg_side from,
+		      uint16_t edge_index, struct rte_mbuf **pkts,
+		      uint64_t pkts_mask,
+		      struct pg_error **errp)
+{
+	struct vtep_state *state = pg_brick_get_state(brick, struct vtep_state);
+
+	if (likely(state->flags == ALL_OPTI))
```
Comment (outscale-jju) [inline src/vtep.c]: todo: remove likely

File: include/packetgraph/vtep.h [hunk 2]
```diff
@@ -55,6 +57,9 @@ void pg_vtep_add_vni(struct pg_brick *brick,
 		  uint32_t vni, uint32_t multicast_ip,
 		  struct pg_error **errp);
 
+void pg_vtep_add_mac(struct pg_brick *brick, uint32_t vni,
+		     struct ether_addr *mac, struct pg_error **errp);
+
```
Comment (outscale-jju) [inline include/packetgraph/vtep.h]: Need header documentation

File: include/packetgraph/vtep.h [hunk 3]
```diff
@@ -31,6 +31,8 @@ enum pg_vtep_flags {
 	ALL_OPTI = NO_COPY | NO_INNERMAC_CKECK
 };
 
+struct ether_addr;
+
```
Comment (outscale-jju) [inline include/packetgraph/vtep.h]: Move this on top

Comment (outscale-mgo) [inline include/packetgraph/vtep.h]: This is just under the enum definition, so it's already on top.

**Pull request merged**